db_stuff.py

```diff
@@ -0,0 +1,44 @@
+import asyncpg
```

Comment:
Rename file away from `db_stuff.py` "stuff" to something more descriptive, like `db_tools.py` or `db_util.py`.